presto-orc/src/test/java/com/facebook/presto/orc/TestListFilter.java

```diff
@@ -0,0 +1,270 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc;
+
+import com.facebook.presto.orc.TupleDomainFilter.BigintRange;
+import com.facebook.presto.orc.TupleDomainFilter.PositionalFilter;
+import com.facebook.presto.orc.reader.ListFilter;
+import com.facebook.presto.spi.Subfield;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.apache.commons.lang3.tuple.Pair;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Random;
+
+import static com.facebook.presto.orc.metadata.OrcType.OrcTypeKind.INT;
+import static com.facebook.presto.orc.metadata.OrcType.OrcTypeKind.LIST;
+import static com.google.common.collect.ImmutableMap.toImmutableMap;
+import static java.lang.String.format;
+import static org.openjdk.jmh.util.Utils.sum;
```

Comment:
A bit weird using jmh in a unit test. Maybe replace with `IntStream.of(...).sum()`

Reply:
Replaced with `Arrays.stream(lengths).sum()`